src/AppBundle/Entity/Feedback.php

```diff
@@ -158,7 +158,8 @@ public function prePersist() {
      * 
      */
     private function setCreated() {
-      // Does nothing
```

Comment:
It's usually a good thing to keep comments like this so that people know someone deliberately chose the function to do nothing.

Reply:
Oh right, I'll restore that immediatlelly

Reply:
Don't worry about it - it's just something to keep in mind for the future.